lighthouse-cli/bin.js

```diff
@@ -122,6 +122,21 @@ async function begin() {
     cliFlags.extraHeaders = JSON.parse(extraHeadersStr);
   }
 
+  if (cliFlags.extraCookies) {
+    // TODO: LH.Flags.extraCookies is actually a string at this point, but needs to be
```

Comment:
major bummer that we'll have a second one of these before we fix the first, this will also actually run into problems with the config-based CLI flags approach.

maybe we could at least extract this to a function to share between the two?

Reply:
well that's the partial fix for the path problem, unless I'm misunderstanding? 

there's not really an obvious fix I see for the "this thing is a complex object that can't be represented by yargs that we want to JSON.parse from the CLI but just use from node", `--cookies-json-string` I guess but that feels 🤢

Reply:
@brendankenny  @patrickhulce I am not pretty sure which fix is required here, as I am Java developer. I assume yargs can't provide you correctly typed JSON value out of the box.

Reply:
You're right on that point @RynatSibahatau! The best we can do and what the comment suggests is that the property name should just be different for the yargs string version.

@brendankenny thoughts on `extraCookiesJsonString` for yargs and `extraCookies` for regular? (or we nix the `extra` prefix entirely I think was your last suggestion?)

Reply:
bump on this @brendankenny

Reply:
we could do `--cookies` for the flag (on the `CliFlags` interface) and `extraCookies` for the setting (on `SharedFlagsSettings`). Or `--cookies-string` and `cookies`.

Whatever it is, we should absolutely use this pattern. It's awkward, but most folks won't really care what the flag is (shortish is probably better, though), and the TODOs that have propagated from this point can only be removed in the future by doing this, so might as well do it before landing :)

Reply:
I vote `--cookies-string` and `--cookies` then! 👍